.github/workflows/flakiness-summary.yml

```diff
@@ -1,33 +1,12 @@
 name: Flaky Test Summary
 
 on:
-  push:
-    branches:
-      - master
-
-env:
-  NUM_RUNS: 1
-  GCS_BUCKET: production_ddp_flow_test_metrics
-  COMMIT_SHA: ${{ github.sha }}
+  schedule:
```

Comment:
what is this script running once a week - level 2 and 3 data upload? Why once/week?